.github/workflows/dependency-check.yml

```
@@ -13,9 +13,12 @@ jobs:
   dependency-sanity-checker:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
-    - run: rustup update
-    - uses: actions/setup-go@v3
-      with:
-        go-version: 1.20.1
-    - run: scripts/check-dependencies.bash
+      - uses: actions/checkout@v3
+      - run: rustup update
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.20.1
+      - run: scripts/check-dependencies.bash
+  validate-rust-git-rev-deps:
+    steps:
+      - uses: stellar/actions/rust-check-git-rev-deps@main
```

The project needs to be checked out.
```suggestion
      - uses: actions/checkout@v3
      - uses: stellar/actions/rust-check-git-rev-deps@main
```

---

sure.